package.json

```diff
@@ -1,75 +1,80 @@
 {
-  "name": "sashay",
-  "description": "A documentation generator for the Percolate API.",
-  "version": "5.0.0",
-  "repository": "https://github.com/percolate/sashay",
-  "license": "CPAL-1.0",
-  "engines": {
-    "node": "12.x.x"
-  },
-  "bin": {
-    "sashay": "./bin/sashay"
-  },
-  "dependencies": {
-    "@babel/core": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/register": "^7.0.0",
-    "ajv": "4.3.0",
-    "babel-loader": "8.0.5",
-    "bluebird": "3.5.3",
-    "classnames": "2.2.5",
-    "commander": "2.19.0",
-    "css-loader": "2.1.1",
-    "expose-loader": "0.7.5",
-    "file-loader": "3.0.1",
-    "fs-extra": "7.0.1",
-    "glob": "7.1.3",
-    "handlebars": "4.3.0",
-    "highlight.js": "9.5.0",
-    "immutable": "3.8.1",
-    "joi": "8.0.5",
-    "json-loader": "0.5.7",
-    "json-schema-ref-parser": "6.1.0",
-    "less": "2.7.1",
-    "less-loader": "4.1.0",
-    "lodash": "4.17.13",
-    "marked": "0.3.18",
-    "mini-css-extract-plugin": "^0.5.0",
-    "raml-parser": "0.8.18",
-    "react": "15.4.2",
-    "react-addons-pure-render-mixin": "15.4.2",
-    "react-dom": "15.4.2",
-    "route-parser": "0.0.5",
-    "style-loader": "0.23.1",
-    "watchpack": "1.6.0",
-    "webfontloader": "1.6.26",
-    "webpack": "4.29.6",
-    "webpack-dev-server": "3.2.1"
-  },
-  "devDependencies": {
-    "@percolate/publisher": "^0.0.8",
-    "chai": "4.2.0",
-    "enzyme": "2.4.1",
-    "eslint": "5.15.1",
-    "eslint-plugin-mocha": "5.3.0",
-    "eslint-plugin-react": "7.12.4",
-    "eslint-plugin-sinon": "0.2.0",
-    "istanbul-instrumenter-loader": "3.0.1",
-    "karma": "4.0.1",
-    "karma-chai": "0.1.0",
-    "karma-chrome-launcher": "2.2.0",
-    "karma-coverage-istanbul-reporter": "^2.0.5",
-    "karma-mocha": "1.3.0",
-    "karma-sinon": "1.0.5",
-    "karma-sourcemap-loader": "0.3.7",
-    "karma-webpack": "3.0.5",
-    "mocha": "6.0.2",
-    "nyc": "^13.3.0",
-    "prettier": "^1.16.4",
-    "react-addons-test-utils": "15.4.2",
-    "semver": "5.3.0",
-    "sinon": "7.2.7",
-    "sinon-chai": "3.3.0"
-  }
+    "name": "sashay",
```

Comment:
Looks like `prettier` changed the indent to 4. Could that have unintended consequences?

Reply:
It shouldn't but reverted just in case ;)